### ISSUE TITLE
`app_installations` appears to be broken

### ISSUE THREAD

**Issue by anthr76** [mineiros-io/terraform-github-repository#122]
I'm not entirely sure here why a set would be preferred over a numbered list, but with this usage:

`app_installations   = [ var.some_number ]`

Produces:

```
The given "for_each" argument value is unsuitable: "for_each" supports maps
and sets of strings, but you have provided a set containing type number.

Error: Invalid for_each set argument

  on .terraform/modules/terraform_github_modules/main.tf line 505, in resource "github_app_installation_repository" "app_installation_repository":
 505:   for_each = var.app_installations
    ├────────────────
    │ var.app_installations is set of number with 1 element
```

**Comment by anthr76** [comment]
Related: https://github.com/mineiros-io/terraform-github-repository/pull/120

**Comment by soerenmartius** [comment]
Thanks for reporting this. Yes - actually this should be a set of strings as an installation ID can be eg. `01234`. I am gonna provide a fix for this later today.

**Comment by soerenmartius** [comment]
We just released [v0.16.2](https://github.com/mineiros-io/terraform-github-repository/releases/tag/v0.16.2) which contains the fix for this issue! Again, thanks for bringing this to our attention!
